### PR TITLE
Pylons documentation

### PR DESCRIPTION
**Pull request by Tittoffer**
**When merged this pull request will:**

- Add documentation for ace3 pylons component. 
- Created pylons.md in wiki/feature

### CODE REVIEW

File: docs/wiki/feature/pylons.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+layout: wiki
+title: Pylons
+description: Adds an interface that allow players to configure aircraft pylons.
```
Comment (jonpas) [inline docs/wiki/feature/pylons.md]: Maybe add "in-game" at the end, to not confuse it with editor pylons.

File: docs/wiki/feature/pylons.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+mod: ace
+version:
+  major: 3
+  minor: 11
+  patch: 0
+---
+
+## 1. Overview
+
+Adds an interface that allow players to configure aircraft pylons. Loadouts can be saved and loaded through interface.
```
Comment (jonpas) [inline docs/wiki/feature/pylons.md]: Same as above, expand further to not have confusion.

File: docs/wiki/feature/pylons.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## 2. Usage
+
+### 2.1 Configure Pylons
+
+To configure pylons you need a `Toolkit` equipped.
```
Comment (jonpas) [inline docs/wiki/feature/pylons.md]: There also has to be a rearm vehicle nearby. Both of those requirements can also be disabled.

File: docs/wiki/feature/pylons.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+- Interact with aircraft using <kbd>⊞ Win</kbd> (ACE3 default key bind `Interaction Key`).
+- Select `Configure Pylons`.
+- Configure your loadout in the menu that appears.
+- Press `APPLY` when done.
+- Rearm the aircraft (See [Rearm Wiki](rearm.md)).
```
Comment (jonpas) [inline docs/wiki/feature/pylons.md]: Lower-case `see`.

File: docs/wiki/feature/pylons.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+To save a custom loadout you first configure it. In the text box above `SAVE`, `LOAD` and `DELETE` you type the loadout name, then press `SAVE`.
+
+To load a custom loadout you select it in the preset pane and press `LOAD` followed by `APPLY`.
+
+To delete a custom loadout you select it in the preset pane and press `DELETE`.
+
+## 3. Dependencies
+
+{% include dependencies_list.md component="" %}
```
Comment (jonpas) [inline docs/wiki/feature/pylons.md]: `component="pylons"`

File: docs/wiki/feature/pylons.md (hunk 4 above)
Comment (654wak654) [reply docs/wiki/feature/pylons.md]: You can turn off the need to do step 5, it's a setting.

File: docs/wiki/feature/pylons.md (hunk 3 above)
Comment (654wak654) [reply docs/wiki/feature/pylons.md]: ☝️  And a require engineer requirement can be enabled.

**Pull request merged**